scripts/exregional_make_ics.sh

```diff
@@ -681,20 +681,134 @@ located in the following directory:
+# NOTES:
+# * The large-scale blending is broken down into 4 major parts
+#   1) chgres_winds: This part rotates the coldstart winds from chgres to the model D-grid.
+#                    It is based on atmos_cubed_sphere/tools/external_ic.F90#L433, and it
+#                    is equivalent to the fv3jedi tool called ColdStartWinds.
+#   2) remap_dwinds: This part vertically remaps the D-grid winds.
+#                    It is based on atmos_cubed_sphere/tools/external_ic.F90#L3485, and it
+#                    is part of the fv3jedi tool called VertRemap.
+#   3) remap_scalar: This part vertically remaps all other variables.
+#                    It is based on atmos_cubed_sphere/tools/external_ic.F90#L2942, and it
+#                    is the other part of the fv3jedi tool called VertRemap.
+#   4) raymond:      This is the actual blending code which uses the raymond filter. The
+#                    raymond filter is a sixth-order tangent low-pass implicit filter
+#                    and can be controlled via the cutoff length scale (Lx).
+#
+# * Currently blended fields: u, v, t, dpres, and sphum
+#     -) Blending only works with GDASENKF (netcdf)
+#
+# * Two RRFS EnKF member files are needed: fv_core and fv_tracer.
+#     -) fv_core contains u, v, t, and dpres
+#     -) fv_tracer contains sphum
+#
+# * Before we can do any blending, the coldstart files from chgres need to be
+#   processed. This includes rotating the winds and vertically remapping all the
+#   variables. The cold start file has u_w, v_w, u_s, and v_s which correspond 
+#   to the D-grid staggering.
+#     -) u_s is the D-grid south tangential wind component (m/s)
+#     -) v_s is the D-grid south face normal wind component (m/s)
+#     -) u_w is the D-grid west  face tangential wind component (m/s)
+#     -) v_w is the D-grid west  face normal wind component (m/s)
+#     -) https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/blob/bdeee64e860c5091da2d169b1f4307ad466eca2c/tools/external_ic.F90
+#     -) https://dtcenter.org/sites/default/files/events/2020/20201105-1300p-fv3-gfdl-1.pdf
+#
+cdate_crnt_fhr_m1=$( date --utc --date "$yyyymmdd $hh UTC - 1 hours" "+%Y%m%d%H" )
+if [[ $DO_ENS_BLENDING == "TRUE" &&
+      $cdate_crnt_fhr -ge ${FIRST_BLENDED_CYCLE_DATE} &&
+      $EXTRN_MDL_NAME_ICS == "GDASENKF" ]]; then
+
+   echo "Blending Starting."
+   ulimit -s unlimited
+   export OMP_STACKSIZE=2G
+   export OMP_NUM_THREADS="96"
+   export FI_OFI_RXM_SAR_LIMIT=3145728
+   export FI_MR_CACHE_MAX_COUNT=0
+   export MPICH_OFI_STARTUP_CONNECT=1
+
+   # Python/F2Py scripts
+   cp_vrfy $SCRIPTSDIR/da_blending_fv3.py .
+   cp_vrfy $SCRIPTSDIR/da_chgres_cold2fv3.py .
+
+   # F2Py shared object files
+   ln_vrfy -sf $EXECDIR/raymond.cpython-38-x86_64-linux-gnu.so .
+   ln_vrfy -sf $EXECDIR/chgres_winds.cpython-38-x86_64-linux-gnu.so .
+   ln_vrfy -sf $EXECDIR/remap_scalar.cpython-38-x86_64-linux-gnu.so .
+   ln_vrfy -sf $EXECDIR/remap_dwinds.cpython-38-x86_64-linux-gnu.so .
```

Comment:
We can remove the "cpython-38-x86_64-linux-gnu" text now

Reply:
@guoqing-noaa Right. Also, EXECDIR needs to be something like LIB64DIR? I don't see the path to lib64 defined anywhere.

Reply:
I had thought about installing those *.so files into bin/.

With this, it will be good that we change `CMAKE_INSTALL_LIBDIR` to `CMAKE_INSTALL_BINDIR` when building your blending code. CMAKE_INSTALL_LIBDIR currently holds intermediate results instead of final results. And I think these four *so files are final results.

If you agree, you may create a PR to rrfs_utl to make changes and we don't need to change EXECDIR here. Thanks!

Reply:
@guoqing-noaa, I added the LIB64DIR variable.